### ISSUE TITLE
OpenMPI not built with gcc-5.2.0

### ISSUE THREAD

**Issue by brucellino** [AAROC/CODE-RADE#81]
the job config needs to be updated.

**Comment by brucellino** [comment]
Updated project definition.

**Comment by brucellino** [comment]
build 25 which happened during the sprint failed. 

v1.10.2 never got off the starting blocks : when untarring the src tarball we get : 
`gzip: stdin: not in gzip format`

because actually : 

```
Resolving www.open-mpi.org... 129.79.39.208
Connecting to www.open-mpi.org|129.79.39.208|:80... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://www.open-mpi.org/software/ompi/v1.1/downloads//openmpi-1.10.2.tar.gz [following]
--2016-06-04 03:56:30--  https://www.open-mpi.org/software/ompi/v1.1/downloads//openmpi-1.10.2.tar.gz
Connecting to www.open-mpi.org|129.79.39.208|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: /404.php [following]
--2016-06-04 03:56:32--  https://www.open-mpi.org/404.php
Connecting to www.open-mpi.org|129.79.39.208|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [text/html]
Saving to: `/repo/src/openmpi/1.10.2/openmpi-1.10.2.tar.gz'

     0K .......... .......                                     55.5K=0.3s

2016-06-04 03:56:34 (55.5 KB/s) - `/repo/src/openmpi/1.10.2/openmpi-1.10.2.tar.gz' saved [18238]
```

**Comment by brucellino** [comment]
the 404 is due to how we express the minor version. instead of resolving to `v1.10`, we resolve to `v1.1` since we're chopping off 3 characters : 
`VERSION_MAJOR=${VERSION:0:3}`

**Comment by brucellino** [comment]
Fixed in d177a2d5e079870c55465ffafc316b6e0d3b4a
